src/ImportClient.ts

```diff
@@ -0,0 +1,131 @@
+import ApiClientBase from './ApiClientBase';
+import { ImportDetailQuery, Authentication, S3AccessKeys } from './Types';
+import { formatDate } from './Util';
+
+export default class ImportClient extends ApiClientBase {
```

Comment:
The import methods seem to be missing content-type as a parameter. We could also make the arguments objects like in some other places.